network-libp2p/src/network.rs

```diff
@@ -686,21 +686,46 @@ impl Network {
                 match event {
                     NimiqEvent::Dht(event) => {
                         match event {
-                            KademliaEvent::OutboundQueryProgressed { id, result, .. } => {
+                            KademliaEvent::OutboundQueryProgressed {
```

Comment:
Unrelated, but we could include the KadmliaEvent type already in NimiqEvent match to reduce the indentations a bit, as they grow quite large.